test/spec/provider/cloud-element-templates/properties/CustomProperties.spec.js

```diff
@@ -302,7 +302,7 @@ describe('provider/cloud-element-templates - CustomProperties', function() {
     });
 
 
-    it('should keep input (non optional)', inject(async function() {
```

Comment:
Apparently by going the route I propose we violate existing contracts (non-optional inputs are always present)?

Looks like I missed on `optional` completely. Will need to assess tomorrow. :see_no_evil: 

---

For reference these are the current (web modeler) templates: https://github.com/camunda/web-modeler/tree/main/webapp/src/App/Pages/Diagram/BpmnJSExtensions/connectorsExtension/.camunda/element-templates.

Reply:
> Ah yes, I thought we already had discussed this, hence "Validation indicates whether the property is needed"

We may, but I may have messed it up (missed the `optional` property), too :see_no_evil:.

Need to check what web modeler is doing, and get back to you. Sorry for that :speak_no_evil:.

Reply:
Moved this to a separate PR (maybe won't fix): https://github.com/bpmn-io/bpmn-js-properties-panel/pull/833.

Reply:
Won't fix.